src/droplet.F

```diff
@@ -689,11 +822,12 @@ subroutine BE_integration(dt,np,pdata,pdata_locind,xf,yf,x3d,y3d, &
          pdata(np,prvpz) = -pdata(np,prvpz)
          pdata_locind(np,3) = undefined_index
       elseif ( z3d .lt. bot ) then
-         !z3d = 1.0e-6  !Just temporary 
-         !pdata(np,pract) = -1.0  !Signal this as no longer alive
-         z3d = bot + (bot-z3d)
-         pdata(np,prvpz) = -pdata(np,prvpz)
+         z3d = 1.0e-6  !Just temporary 
+         pdata(np,pract) = -1.0  !Signal this as no longer alive
+         !z3d = bot + (bot-z3d)
+         !pdata(np,prvpz) = -pdata(np,prvpz)
          pdata_locind(np,3) = undefined_index
+         num_fallout = num_fallout + 1
```

Comment:
For my own reference: `num_fallout` needs to be put on the `reduction` clause for GPU code.